app/src/modules/settings/routes/roles/item/components/permissions-overview-header.vue

```diff
@@ -9,22 +9,19 @@
 	</div>
 </template>
 
-<script lang="ts">
+<script lang="ts" setup>
 import { useI18n } from 'vue-i18n';
-import { defineComponent } from 'vue';
 
-export default defineComponent({
-	setup() {
-		const { t } = useI18n();
-		return { t };
-	},
-});
+const { t } = useI18n();
 </script>
 
 <style lang="scss" scoped>
 .permissions-overview-header {
 	--v-icon-color: var(--foreground-subdued);
 
+	position: sticky;
+	top: 60px; // height of header bar minus 1px to avoid gaps
```

Comment:
I was basing it from the height value over here:

https://github.com/directus/directus/blob/fc9190e76e47dc30c694d112c3f768f202cabb35/app/src/views/private/components/header-bar/header-bar.vue#L99

perhaps it is not present in the current code? Can't seem to find it but I was only looking up `60px` or `61px` across the app.

Though I did also find the calendar layout referring to `61px` which should (presumably) also be the header bar height, so it would indeed be helpful to have that consolidated as a variable if it doesn't exist. Should I add it?

https://github.com/directus/directus/blob/fc9190e76e47dc30c694d112c3f768f202cabb35/app/src/layouts/calendar/calendar.vue#L43

Reply:
Interesting! We probably should 👍🏻 I was convinced there was already a way to get that data, but 🤷🏻

Reply:
Added!